tests/testthat/test-driver-oracle.R

```diff
@@ -0,0 +1,30 @@
+test_that("Oracle", {
+  DBItest::make_context(
+    odbc(),
+    test_connection_string("ORACLE"),
+    tweaks = DBItest::tweaks(temporary_tables = FALSE, placeholder_pattern = "?"),
+    name = "PostgreSQL"
+  )
+
+  # - Long/outstanding issue with batch inserting
+  # date/datetime for Oracle.  See for example
+  # #349, #350, #391
+  # - There also looks like there are issues related
+  # to binary elements of size zero.
+  # - Finally, no boolean in Oracle prior to 23
+  test_roundtrip(columns = c("time", "date", "datetime", "binary", "logical"))
+
+  local({
+    # Test custom dbExistsTable implementation for
+    # Oracle
+    con <- DBItest:::connect(DBItest:::get_default_context())
```

Comment:
I think it would be better to make this an individual test, using `DBI::dbConnect(odbc::odbc(), .connection_string = test_connection_string("ORACLE")))` (or some new wrapper thereof). Then, later, we can convert the other tests to the same structure.